reference/ipset.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: IP Sets
+---
+
+## Overview
+
+An ipSet reserves a public IP address for each location configured within a Global Virtual Cloud
+([GVC](gvc)). The public IP address is stored in the object’s status. When a new location is added to a
+[GVC](gvc), a new IP address is automatically created if one does not already exist for that location.
+To prevent ongoing charges, IP addresses must be explicitly released once they are no longer needed.
+
+## Binding
+
+An IP Set can be linked to a [workload](workload) or [GVC](gvc) through its `spec.link` property.
+When linking to a [GVC](gvc), [dedicated load balancing](gvc#dedicated-load-balancer) must be
+enabled for the GVC. When linking to a [workload](workload), the workload must have
+[direct load balancer](workload/direct-load-balancer) enabled. Alternatively, a workload can link
+to an IP Set via its [direct load balancer](workload/direct-load-balancer). If both the workload
+and IP Set are linked, the IP Set will only function if the cross-linking is properly configured.
```

Comment:
The IP Set will only be used when it is cross linked. For example, if the IP Set target is a workload, the workload must be configured with the ipSet property.